docs/user-event/api-pointer.mdx

```diff
@@ -72,7 +72,7 @@ pointer([
   // touch the screen at element1
   {keys: '[TouchA>]', target: element1},
   // move the touch pointer to element2
-  {pointerName: 'TouchA', target: element2},
+  {pointerName: 'touch', target: element2},
```

Comment:
If I'm not mistaken, then existing docs are correct.
`TouchA` is used to move the touch (first pointer) from element1 to element2.
Feel free to correct me @ph-fritsche

Reply:
This is indeed the correct behavior, the name of the pointer is `TouchA` as can be seen here:
https://github.com/testing-library/user-event/blob/main/src/pointer/keyMap.ts#L7

I'm closing this PR :)